smart_control/simulator/building_radiation_test.py

```diff
@@ -71,2613 +77,8 @@ def test_interior_radiative_heat_transfer(self):
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-          ],
-          [
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.02844378,
-              0.02844378,
-              0.02844378,
-              0.02844378,
-              0.02844378,
-              0.02844378,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-          ],
-          [
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.02844378,
-              0.02844378,
-              0.02844378,
-              0.02844378,
-              0.02844378,
-              0.02844378,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0510298,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-              0.0,
-          ],
-      ])
-
-      assert_array_almost_equal(
-          np.round(self.building.interior_wall_VF, 5),
-          np.round(expected_interior_wall_vf, 5),
-      )
+      props = self.building.interior_wall_vf
+      assert_data_file_values_equal(props, "expected_interior_wall_vf.csv")
```

Comment:
Love that we moved these expected values into a CSV file. Makes the tests much easier to read 🙌 ❤️